emcc.py

```diff
@@ -2876,12 +2723,6 @@ def do_binaryen(target, asm_target, options, memfile, wasm_binary_target,
     building.save_intermediate(wasm_binary_target, 'pre-ctors.wasm')
     building.eval_ctors(final, wasm_binary_target, binaryen_bin, debug_info=intermediate_debug_info)
 
-  # after generating the wasm, do some final operations
-  if shared.Settings.SIDE_MODULE and not shared.Settings.WASM_BACKEND:
```

Comment:
Could you leave this in but maybe commented or with a TODO?

Reply:
Sure, done.